RELEASE/scripts/sl_ascend.ash

```diff
@@ -11369,7 +11366,7 @@ boolean LX_handleSpookyravenFirstFloor()
 			expectPool += 3;
 		}
 
-		if(!possessEquipment($item[Pool Cue]) && !posessEquipment($item[7964]) && !posessEquipment($item[2268]) && !posessEquipment($item[7961]))
+		if(!possessEquipment($item[Pool Cue]) && !possessEquipment($item[7964]) && !possessEquipment($item[2268]) && !possessEquipment($item[7961]))
```

Comment:
Maybe try doing `ashq import<sl_ascend.ash>` before committing next time 😛 

Or better yet (I just did this myself) `alias testsl => ashq import<sl_ascend.ash> print("All is well.")` and then just `testsl` before committing. Only good for catching compilation errors, but better than nothing.

Reply:
:(